src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -30,15 +30,15 @@ private class FindLiteralsProgressAdapter : IStreamingFindLiteralReferencesProgr
                 _definition = definition;
             }
 
-            public async Task OnReferenceFoundAsync(Document document, TextSpan span)
+            public async Task OnReferenceFoundAsync(Document document, TextSpan span, CancellationToken cancellationToken)
             {
                 var documentSpan = await ClassifiedSpansAndHighlightSpan.GetClassifiedDocumentSpanAsync(
                     document, span, _context.CancellationToken).ConfigureAwait(false);
```

Comment:
@heejaechang That one actually can't be removed, since `ServiceHubServiceBase` provides one of the cancellation token sources. But I renamed the accessor field and updated all uses which did not depend specifically on that source.